vendor/bat-native-ledger/src/bat/ledger/internal/media/twitter.cc

```diff
@@ -63,6 +67,116 @@ std::string MediaTwitter::GetMediaKey(const std::string& screen_name) {
   return (std::string)TWITTER_MEDIA_TYPE + "_" + screen_name;
 }
 
+// static
+std::string MediaTwitter::GetUserNameFromUrl(const std::string& url) {
+  if (url.empty()) {
+    return std::string();
+  }
+
+  std::vector<std::string> parts = base::SplitString(
+      url, "/", base::TRIM_WHITESPACE, base::SPLIT_WANT_NONEMPTY);
+
+  if (parts.size() > 0) {
+    return parts[0];
+  }
+
+  return std::string();
+}
+
+// static
+bool MediaTwitter::IsExcludedPath(const std::string& path) {
+  std::vector<std::string> paths({
+      "/",
+      "/settings",
+      "/explore",
+      "/notifications",
+      "/messages",
+      "/logout",
+      "/search",
+      "/about",
+      "/tos",
+      "/privacy",
+      "/home"
+    });
+
+  for (std::string str_path : paths) {
+    if (str_path == path) {
+      return true;
+    }
+  }
+
+  std::vector<std::string> patterns({
+    "/i/",
+    "/account/",
+    "/compose/",
+    "/who_to_follow/",
+    "/hashtag/",
+    "/settings/"
+  });
+
+  for (std::string str_path : patterns) {
+    if (base::StartsWith(path,
+                         str_path,
+                         base::CompareCase::INSENSITIVE_ASCII)) {
+      return true;
+    }
+  }
+
+
+  return false;
+}
+
+// static
+std::string MediaTwitter::GerUserId(const std::string& response) {
```

Comment:
Typo here? Is it supposed to be `GetUserId`?

Reply:
done